app/views/home/start.html.erb

```diff
@@ -16,7 +16,7 @@
 
       <% if save_and_return_enabled? %>
         <li>
-          <a href="/sign_in" class="task-link">Return to an existing appeal or application</a>
+          <%= link_to 'Return to an existing appeal or application', new_user_session_path, class: 'task-link ga-pageLink' %>
```

Comment:
Can we have this text in the translation file, please? I think Josh might have done some work on this already.